tracer/src/Datadog.Trace/IAST/VulnerabilityType.cs

```diff
@@ -0,0 +1,13 @@
+// <copyright file="VulnerabilityType.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+namespace Datadog.Trace.Iast
+{
+    internal enum VulnerabilityType
+    {
+        WEAK_CIPHER,
+        WEAK_HASH
```

Comment:
Should we try to stick to C# conventions and name these `WeakCipher` and `WeakHash`? Unless these names are used a strings when serializing (which may be a separate performance issue).

Reply:
Cool. Done!